docs/pages/usage/refund_punish.mdx

```diff
@@ -1,24 +1,41 @@
+If the _maker_ has not been able to redeem the Bitcoin within ~12 hours (72 Bitcoin blocks) from the start of the swap, the swap will be cancelled.
+This is done by either you or the _maker_ by publishing a special Bitcoin transaction called the `Bitcoin cancel transaction`, or `tx_cancel`.
+As soon as this transaction is included in the Bitcoin blockchain, the swap is locked in a state where only the [_refund_](#refund) and [_punish_](#punish) paths can be activated. 
+The _happy path_ where you redeem the Monero is no longer possible.
 
 ## Refund
 
-As soon as the swap is cancelled, you can refund your Bitcoin. This is done by publishing the `Bitcoin Refund Transaction` on the Bitcoin blockchain.
-If this is done within 24 hours (144 Bitcoin blocks) from the inclusion of the `Bitcoin Cancel Transaction`, you will get your Bitcoin back.
-If you do not refund your Bitcoin within this time frame, the _maker_ can punish you. This is a security measure to ensure that you do not cancel the swap and then refuse to refund your Bitcoin which would result in the _maker_ losing their Monero.
+As soon as the swap is cancelled, you can (and should) refund your Bitcoin. 
+This is done by publishing the `Bitcoin refund transaction`, or `tx_refund`.
+If this is done within 24 hours (144 Bitcoin blocks) from the confirmation of the `Bitcoin cancel transaction`, you will get your Bitcoin back.
+If you do not refund your Bitcoin within this time frame, the _maker_ can punish you. 
+This is a security measure to ensure that malicious actors can't refund their Bitcoin while the _maker_ loses their Monero.
 
 ## Punish
 
-If you do not refund your Bitcoin within 24 hours (144 Bitcoin blocks) from the inclusion of the `Bitcoin Cancel Transaction`, the _maker_ will _punish_ you. This will result in the _maker_ taking your Bitcoin as a penalty for not refunding it in time.
-Even if this state is reached and the _maker_ has punished you, there's still hope to redeem the Monero. The _maker_ can choose to allow you to redeem the Monero by transmitting a secret key to you. This however is at the discretion of the _maker_ and they are not obligated to do so.
\ No newline at end of file
+If you do not refund your Bitcoin within 24 hours (144 Bitcoin blocks) from the confirmation of the `Bitcoin cancel transaction`, the _maker_ will _punish_ you. 
+This will result in the _maker_ taking your Bitcoin as a penalty for not refunding it in time.
+Even if this state is reached and the _maker_ has punished you, there's still hope to redeem the Monero. 
+The _maker_ can choose to allow you to redeem the Monero by transmitting a secret key to you. 
+This however is at the discretion of the _maker_ and they are not obligated to do so.
```

Comment:
Clarify that this will be possible in most cases (if maker is online) (less scary)